scripts/migration/migrate-linux.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/sh
+
+echo -e "********************************************"
+echo -e "      Ferdium User Data Migration Tool      "
+echo -e "          WARNING: Linux OS Only!           "
+echo -e "********************************************"
+echo -e "\n\nThis tool migrates your user data from Ferdi to Ferdium.\n\n"
+read -p "Do you want to proceed? (y/N) " confirm
+
+case "$confirm" in
+    y|Y|[yY][eE][sS] ) echo -e "Starting...";;
+    n|N|[nN][oO] ) exit 1;;
+    * ) exit 1;;
+esac
+
+FERDI_PATH="$HOME/.config/Ferdi"
+FERDIUM_PATH="$HOME/.config/Ferdium"
+
+if [ -d $FERDIUM_PATH ]; then
+    echo -e "\nPath $FERDIUM_PATH exist, making a backup\n"
+    if ! mv -vf $FERDIUM_PATH $FERDIUM_PATH.bak;then
+        read -p "A previous backup already exists at $FERDIUM_PATH.bak. do you want to remove it? (y/N) " confirm
+        echo
+        case "$confirm" in
+            y|Y|[yY][eE][sS] ) echo -e "Deleting...";;
+            n|N|[nN][oO] ) exit 1;;
+            * ) exit 1;;
+        esac
+        echo
+        rm -rf $FERDIUM_PATH.bak
+        mv -vf $FERDIUM_PATH $FERDIUM_PATH.bak
+    fi
+fi
+
+if mv -vf $FERDI_PATH $FERDIUM_PATH; then
+    cp -rf $FERDIUM_PATH $FERDI_PATH
```

Comment:
If you are anyways moving the whole folder with the `mv`, why do we need a `cp -R`?

Reply:
this is for if anyone needs to use the Ferdi app again. since i only need the output of mv -vf for the folder for uniformity. but if it bothers anyone i'll gladly remove it

Reply:
yes - i dont think we need to support all scenarios. For our usecase, its only move from Ferdi to Ferdium

Reply:
Alright then, i'll made changes accordingly